app/main.py

```diff
@@ -1,8 +1,29 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    person_list = []
+
+    for i in range(len(people)):
+        person_list.append(Person(people[i]["name"], people[i]["age"]))
+
+    for i in range(len(people)):
```

Comment:
U can iterate by person here, not by index. Don't forget to use descriptive name